SwiftCheck/Test.swift

```diff
@@ -246,18 +252,18 @@ public func runATest(st: State)(f: (StdGen -> Int -> Prop)) -> Either<Result, St
 					st2.randomSeed = rnd2
 					st2.collected = [stamp] + st.collected
 					st2.expectedFailure = expect
```

Comment:
This mutation stuff needs to die, but I'm not sure how...

I would kill for `struct { member = new }` notation